db/migrate/20250715093722_change_invalid_claims_status.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeInvalidClaimsStatus < ActiveRecord::Migration[8.0]
+  def up
+    accredited_providers = Claims::Provider.accredited
+    invalid_claims = Claims::Claim.where.not(provider: accredited_providers)
+
+    # We want this to be reflected in Big Query, so we're using callbacks despite the performance hit
+    invalid_claims.find_each { |claim| claim.update!(status: :invalid_provider) }
+
+    Claims::Claim::InvalidProviderNotification.call
```

Comment:
Do we really want migrations sending emails? Since this is a one-off thing, there might be a case for doing it in a rake task